pkgs/development/libraries/rapidjson/default.nix

```diff
@@ -38,6 +38,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DGTEST_SOURCE_DIR=${gtest.dev}/include"
+  ] ++ lib.optionals stdenv.hostPlatform.isStatic [
+    "-DRAPIDJSON_BUILD_TESTS=OFF"
```

Comment:
I'd suggest doing `doCheck = !stdenv.hostPlatform.isStatic` down there :point_down: and making this

```
] ++ lib.optionals doCheck [
```

Reply:
(ah yes the inverse quite right)